src/automata/RequestResponseRequester.cpp

```diff
@@ -110,31 +104,35 @@ void RequestResponseRequester::onNextFrame(Frame_ERROR&& frame) {
 }
 
 void RequestResponseRequester::onNextFrame(Frame_RESPONSE&& frame) {
-  bool end = false;
   switch (state_) {
     case State::NEW:
       // Cannot receive a frame before sending the initial request.
       CHECK(false);
```

Comment:
Are these absolutely only internal invariants? e.g. data corruption on the wire could end up here?